application/src/main/java/core/application/config/PrivateConfig.java

```diff
@@ -4,7 +4,7 @@
 import org.springframework.context.annotation.PropertySource;
 
 @Configuration
-@PropertySource("classpath:privates/private.properties")
+@PropertySource("classpath:application.properties")
```

Comment:
민감 정보 숨기기 위한 설정이니 그대로 두셔야 합니다.

```suggestion
@PropertySource("classpath:privates/private.properties")
```